app/frontend/tests/unit/components/designer/settings/FormEventStreamSettings.spec.js

```diff
@@ -8,10 +8,6 @@ import { useFormStore } from '~/store/form';
 import FormEventStreamSettings from '~/components/designer/settings/FormEventStreamSettings.vue';
 
 describe('FormEventStreamSettings.vue', () => {
-  const crypto = require('crypto').webcrypto;
-  // Shims the crypto property onto global
-  global.crypto = crypto;
-
```

Comment:
This was failing with an error about global not having setters. Must be something new in Node.js v20. Seems to run fine without it, so Node.js's global.crypto probably comes by default now.